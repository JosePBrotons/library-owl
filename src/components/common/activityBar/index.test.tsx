import React from 'react';
import renderer from 'react-test-renderer';
import ActivityBar from '.';

test('Renders ActivityBar component correctly', () => {
    const props = {
        icon: 'wifi-off',
        text: 'No connection',
    };
    const activityBarCmp = renderer.create(<ActivityBar {...props} />).toJSON();
    expect(activityBarCmp).toMatchSnapshot();
});
