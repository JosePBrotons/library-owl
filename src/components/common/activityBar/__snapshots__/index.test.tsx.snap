// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders ActivityBar component correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(69, 69, 69, 0.5)",
      "bottom": 0,
      "flexDirection": "row",
      "height": 40,
      "justifyContent": "center",
      "position": "absolute",
      "width": "100%",
      "zIndex": 100,
    }
  }
>
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": "rgba(255, 255, 255, 1)",
          "fontSize": 20,
        },
        undefined,
        Object {
          "fontFamily": "Feather",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    ïˆ“
  </Text>
  <Text
    style={
      Object {
        "color": "rgba(255, 255, 255, 1)",
        "fontFamily": "Nunito-Regular",
        "fontSize": 16,
        "lineHeight": 22,
        "marginHorizontal": 10,
      }
    }
  >
    No connection
  </Text>
</View>
`;
